config.js

```diff
@@ -11,12 +11,142 @@ System.config({
 System.config({
```

Comment:
> WF doesn't have any transitive dependencies, its flat I think.

[WF config.js](https://workflow.gutools.co.uk/assets/config.js) seems to have just a few transitive dependencies on npm:process and github:jspm/nodelibs-process. But I wouldn't rely on that going forward (e.g. pandular has a dep).

I think the config.js also encodes the information about any version conflict resolution.

I know there is some `--lock` flag when jspm installing stuff but I haven't looked into what it does.

It may be worth defining the best practice around this now, and I'm not sure if there's any way to get away without committing the config.js...?

> Q: is config.js read from a dependency when a package is loaded as a dependency?

No.